alfred-telemetry-platform/src/main/resources/alfresco/module/alfred-telemetry-platform/alfresco-global.properties

```diff
@@ -60,3 +60,4 @@ alfred.telemetry.binder.solr.sharding.cronexpression=0/10 * * * * ?
 alfred.telemetry.binder.solr.sharding.enabled=true
 alfred.telemetry.binder.solr.sharding.floc.id.enabled=true
 alfred.telemetry.binder.solr.tracking.enabled=true
+alfred.telemetry.binder.alfresco.node.enabled=true
```

Comment:
One last change request from me: can you change that into `.alfresco-node.` instead ?

Why: naming convention, so the admin-console can automatically adapt to new binders and show the config options.

So the config property structure should follow this convention:
`alfred.telemetry.binder.<binder-name>.<property>[.<optional-sub-property>]=<value>`

(Yes, the `solr.sharding` and `solr.tracking` metrics don't follow this convention right now, we'll need to fix that in a different PR too)